backend/README.md

```diff
@@ -1,203 +1,403 @@
-# FastAPI Project - Backend
+### FastAPI PROJECT! Backend guide 
+
+## Project Technologies 
+
+Backend
+⚡ [**FastAPI**]: Framework for the Python backend API.(https://fastapi.tiangolo.com)
+🧰 [SQLModel]: ORM for SQL database interactions.(https://sqlmodel.tiangolo.com)
+🔍 [Pydantic]: Data validation & settings management used by FastAPI.(https://docs.pydantic.dev)
+💾 [PostgreSQL]: SQL database.(https://www.postgresql.org)
+🔒 Secure password hashing by default.
+🔑 JWT (JSON Web Token) authentication.
+📫 Email-based password recovery.
+✅ Tests with [Pytest].(https://pytest.org)
+[Docker]:A platform for developing, shipping, and running applications in isolated containers.
+(https://www.docker.com/)
+[Poetry]: A tool for Python package and environment management, facilitating dependency management and packaging. (https://python-poetry.org/) 
+* Backend docs: [backend/README.md](./backend/README.md).
+
+
+# Set Up the Backend
+1. Navigate to the backend folder
+```bash
+cd backend
+```
+2. Install dependencies and set up the environment:
+```bash
+# For Python dependencies
+pip install -r requirements.txt
```

Comment:
We don't have a requirements.txt file in the repo. All of the configuration is in `pyproject.toml` and managed by poetry